approvals/approvals/api.py

```diff
@@ -15,22 +15,22 @@
 def get_approval_roles(doc: Document, method: str | None = None):
```

Comment:
@agritheory I made these changes cause we need the `Document Approval Rule` name in frontend to pass down to the rejection method to look up the `primary_rejection_user`.

The problem is we only have this information when a Document Approval Rule applies and not for manual assignations (`add_user_approval`).